File: mario.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int main(void)
{
    int height, i, j;
    //check inserted value
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);
    for (i = 1; i <= height; i++)
    {
        for (j = 1; j <= height - i; j++)
        {
            //for spaces
            printf(" ");
        }
        for (int l = 1; l <= i; l++)
        {
            printf("#");
        }
        //for columns
        printf("\n");
    }

}

File: mario.py
from cs50 import get_int


def main():
    while True:
        height = get_int("Height: ")
        if height > 0 and height < 9:
            break
    pyramid(height, 1)


def pyramid(height, c):
    if height == 0:
        quit()
    space = height - 1
    print(" " * space, end="")
    print("#" * (c), end="")
    print("")
    c += 1
    pyramid(space, c)


main()